views/celebrities/celebrities.hbs

```diff
@@ -0,0 +1,12 @@
+<h1>Celebrity</h1>
+
+{{#each celebrity}}
```

Comment:
In the `celebrities.routes.js` you're sending the celebrities information to the view as `{celebrities}`:

```
res. render (`celebrities/celebrities`, {celebrities});
```

This means that if you want to display this information inside the `celebrities.hbs` file you need to reference the same name that you're passing in the route, so instead of `{{#each celebrity}}` it should be `{{#each celebrities}}`:

```
<h1>Celebrity</h1>

{{#each celebrities}}
  <h2>{{name}}</h2>
  <p>{{occupation}}</p>
  <p>{{catchPhrase}}</p>
  <a href="/celebrities{{_id}}/edit">Edit</a>

  <form action="/celebrities/{{_id}}/delete" method="post">
    <button type="submit">Delete Celebrity</button>
  </form>
{{/each}}
```